reference/5.1/Microsoft.PowerShell.Core/About/about_Character_Encoding.md

```diff
@@ -0,0 +1,230 @@
+8-bit values to represent the characters used in a specific language or
+geographical region.
+
+PowerShell uses a Unicode character set by default. However, several cmdlets
+have an **Encoding** parameter that changes the encoding to a different
+character set. This parameter allows you to choose the specific the character
+encoding you need for interoperability with other systems and applications.
+
+The following cmdlets have the **Encoding** parameter:
+
+- Microsoft.PowerShell.Management
+  - Add-Content
+  - Get-Content
+  - Set-Content
+- Microsoft.PowerShell.Utility
+  - Export-Clixml
+  - Export-Csv
+  - Export-PSSession
+  - Format-Hex
+  - Import-Csv
+  - Out-File
+  - Select-String
+  - Send-MailMessage
+
+## The byte-order-mark
+
+The byte-order-mark (BOM) is a _Unicode signature_ in the first few bytes of a
+file or text stream that indicate which Unicode encoding is being used. For
+more information, see the
+[Byte order mark](https://en.wikipedia.org/wiki/Byte_order_mark) article in
+Wikipedia.
+
+In Windows PowerShell, any Unicode encoding, except `Utf7`, always creates a
+BOM. PowerShell Core defaults to `utf8NoBOM` for all text output.
+
+For best overall compatibility, avoid using BOMs in UTF-8 files. Unix platforms
+and Unix-heritage utilities also used on Windows Platforms don't support BOMs.
+
+Similarly, `utf7` encoding should be avoided. UTF-7 is not a standard Unicode
+encoding and is written without a BOM in all versions of PowerShell.
+
+Creating PowerShell scripts on a Unix-like platform or using a cross-platform
+editor on Windows, such as Visual Studio Code, results in a file encoded using
+`UTF8NoBOM`. These files work fine on PowerShell Core, but may break on Windows
+PowerShell if the file contains non-Ascii characters.
+
+If you do need to use non-Ascii characters in your scripts, save them as UTF-8
+with BOM. Without the BOM, Windows PowerShell misinterprets your script as
+being encoded in the legacy "ANSI" codepage. Conversely, files that do have the
+UTF-8 BOM can be problematic on Unix-like platforms. Many Unix tools such as
+`cat`, `sed`, `awk`, and some editors such as `gedit` don't know how to treat
+the BOM.
+
+## Character encoding in Windows PowerShell
+
+In PowerShell 5.1, the **Encoding** parameter supports the following values:
+
+- `Ascii` Uses Ascii (7-bit) character set.
+- `BigEndianUnicode` Uses UTF-16 with the big-endian byte order.
+- `BigEndianUTF32` Uses UTF-32 with the big-endian byte order.
+- `Byte` Encodes a set of characters into a sequence of bytes.
+- `Default` Uses the encoding that corresponds to the system's active code page
+  (usually ANSI).
+- `Oem` Uses the encoding that corresponds to the system's current OEM code
+  page.
+- `String` Same as `Unicode`.
+- `Unicode` Uses UTF-16 with the little-endian byte order.
+- `Unknown` Same as `Unicode`.
+- `UTF7` Uses UTF-7.
+- `UTF8` Uses UTF-8.
+- `UTF32` Uses UTF-32 with the little-endian byte order.
```

Comment:
Small nit, but I'd put UTF32 between Unknown and UTF7 because that is where it is in the tab ordering.

Reply:
Fixed